models/textMessages.js

```diff
@@ -0,0 +1,7 @@
+const TextMessages = (connection, Sequelize) => connection.define('message', {
```

Comment:
Make sure the tables Contacts and TextMessages are linked by a many to many relation
Some of the fields the new table may have are
`contact_id, text_message_id, sent_date`